README.md

```diff
@@ -37,48 +37,48 @@ This project chooses best practices as defaults:
 | Declarative test definition                                                               | ✔️       | ✔️                                                           | ✔️                                                                |
 | Monadic test definition                                                                   | ✔️       | ✔️                                                           | ✖️                                                                |
```

Comment:
I left this as ✖️ because it's really just not applicable to `tasty` using a pure structure for defining tests.
You'd have to make each test file in your suite have some `IO TestTree` function and then something to combine them to make this work. Going deeper into these while still allowing for IO is going to be even more of a hastle.